war/src/main/less/base/layout-commons.less

```diff
@@ -1,7 +1,6 @@
 @import "../abstracts/theme.less";
 
 html {
-  position: relative;
```

Comment:
unrelated?

Reply:
Having `position: relative` on `html` caused the new tooltips to break, so removed it.